lib/minheap.js

```diff
@@ -37,30 +47,58 @@ class MinHeap {
     return output;
   }
 
-  // This method returns true if the heap is empty
-  // Time complexity: ?
-  // Space complexity: ?
+  // Time complexity: O(1)
+  // Space complexity: O(1)
   isEmpty() {
-    throw new Error("Method not implemented yet...");
+    return this.store.length === 0;
   }
 
-  // This helper method takes an index and
-  //  moves it up the heap, if it is less than it's parent node.
-  //  It could be **very** helpful for the add method.
-  // Time complexity: ?
-  // Space complexity: ?
+  // This helper method takes an index and moves it up the heap
+  // if it is less than its parent node.
+  // Time complexity: O(log n), where n is the number of levels in the heap
+  // Space complexity: O(1)
   heapUp(index) {
```

Comment:
Since you're doing `heapUp` recursively you have some space complexity here.  The space complexity should be O(log n).